src/main/java/com/scottlogic/librarygradproject/BookRepository.java

```diff
@@ -47,7 +47,8 @@ public Book get(int id) {
     @Override
     public void add(Book book) {
         Book newBook = this.validateBook(book);
-        newBook.setId(bookCollection.size());
+        newBook.setId(id);
+        id++;
```

Comment:
Ha! Knew we'd have to do this at some point, could be call it nextId instead? Maybe a little clearer

Reply:
this is going to be overwritten when we implement the database anyway, just fixing it for now so Dev does not contain bugs